hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -1183,12 +1183,17 @@ private Constants() {
 
   /**
    * Flag for create performance.
-   * This is *not* a configuration option; it is for use in the
-   * {code createFile()} builder.
+   * This can be set in the {code createFile()} builder.
```

Comment:
can we add what "create performance" means or change it to a more meaningful name, such as fs.s3a.create.skip.file.existence.check or fs.s3a.create.overwrite-if-exist?

Reply:
i'd rather not because its the existing openFile option. And it gives us options to do more later, if we can think of

Reply:
sure, understood. we can keep this option. But can we add a comment about what it does today? 

I don't get what it meant by reading "create performance".

Reply:
should be covered in the openfile docs